app/views/shared/_single_page_notification_button.html.erb

```diff
@@ -1,6 +1,9 @@
+<% skip_account = skip_account || nil %>
```

Comment:
Is nil the right default here? Would we want to default to false?

Reply:
I assumed it would be nil or "true" when I shipped [this](https://github.com/alphagov/govuk_publishing_components/pull/3229/files), but without any validation that those are the only two options, that is a bug waiting to happen isn't it!!! Because putting "false" as the `skip account` value passed to the component, will make [this](https://github.com/alphagov/govuk_publishing_components/blob/main/lib/govuk_publishing_components/presenters/single_page_notification_button_helper.rb#L64) line select the wrong form action. 

So I will update this line here to default to "false"
And I will update the component so it expects either "true" or "false".

Reply:
https://github.com/alphagov/govuk_publishing_components/pull/3471